jpo-ode-svcs/src/main/resources/application.yaml

```diff
@@ -13,3 +22,88 @@ ode:
       - "topic.OdeBsmTxPojo"
       - "topic.OdeBsmDuringEventPojo"
       - "topic.OdeTimBroadcastPojo"
+    topics:
+      asn1:
+        decoder-input: "topic.Asn1DecoderInput"
+        decoder-output: "topic.Asn1DecoderOutput"
+        encoder-input: "topic.Asn1EncoderInput"
+        encoder-output: "topic.Asn1EncoderOutput"
+      json:
+        bsm: "topic.OdeBsmJson"
+        map: "topic.OdeMapJson"
+        psm: "topic.OdePsmJson"
+        spat: "topic.OdeSpatJson"
+        srm: "topic.OdeSrmJson"
+        ssm: "topic.OdeSsmJson"
+        tim: "topic.OdeTimJson"
+        driver-alert: "topic.OdeDriverAlertJson"
+        tim-broadcast: "topic.OdeTimBroadcastJson"
+        bsm-filtered: "topic.FilteredOdeBsmJson"
+        spat-filtered: "topic.FilteredOdeSpatJson"
+        tim-filtered: "topic.FilteredOdeTimJson"
+        tim-cert-expiration: "topic.OdeTIMCertExpirationTimeJson"
+        dn-message: "topic.OdeDNMsgJson"
+        rx-tim: "topic.OdeTimRxJson"
+        rx-spat: "topic.OdeSpatRxJson"
+        j2735-tim-broadcast: "topic.J2735TimBroadcastJson"
+      pojo:
+        bsm: "topic.OdeBsmPojo"
+        spat: "topic.OdeSpatPojo"
+        ssm: "topic.OdeSsmPojo"
+        tim-broadcast: "topic.OdeTimBroadcastPojo"
+        bsm-during-event: "topic.OdeBsmDuringEventPojo"
+        rx-bsm: "topic.OdeBsmRxPojo"
+        rx-spat: "topic.OdeSpatRxPojo"
+        tx-bsm: "topic.OdeBsmTxPojo"
+        tx-map: "topic.OdeMapTxPojo"
+        tx-psm: "topic.OdePsmTxPojo"
+        tx-spat: "topic.OdeSpatTxPojo"
+        tx-srm: "topic.OdeSrmTxPojo"
+      raw-encoded-json:
+        bsm: "topic.OdeRawEncodedBSMJson"
+        map: "topic.OdeRawEncodedMAPJson"
+        psm: "topic.OdeRawEncodedPSMJson"
+        spat: "topic.OdeRawEncodedSPATJson"
+        srm: "topic.OdeRawEncodedSRMJson"
+        ssm: "topic.OdeRawEncodedSSMJson"
+        tim: "topic.OdeRawEncodedTIMJson"
+      sdx-depositor:
+        input: "topic.SDWDepositorInput"
+      file:
+        filtered-output: "/topic/filtered_messages"
+        unfiltered-output: "/topic/unfiltered_messages"
+  receivers:
+    generic:
+      buffer-size: 2000
+      receiver-port: 44990
+    bsm:
+      buffer-size: 500
+      receiver-port: 46800
+    map:
+      buffer-size: 2048
+      receiver-port: 44920
+    psm:
+      buffer-size: 500
+      receiver-port: 44940
+    spat:
+      buffer-size: 500
+      receiver-port: 44910
+    srm:
+      buffer-size: 500
+      receiver-port: 44930
+    ssm:
+      buffer-size: 500
+      receiver-port: 44900
+    tim:
+      buffer-size: 500
+      receiver-port: 47900
+  file-importer:
+    buffer-size: 500
+    backup-dir: "backup"
+    failed-dir: "failed"
+    time-period: 5
+    upload-location-root: "uploads"
+    obu-log-upload-location: "bsmlog"
+  tim-ingest-monitoring:
+    tracking-enabled: ${TIM_INGEST_TRACKING_ENABLED}
+    interval: ${TIM_INGEST_MONITORING_INTERVAL}
```

Comment:
I have corrected the names of the environment variables for the tim-ingest-monitoring in 

The default value is a little more complex. In short, there's already a default value set [here ](https://github.com/CDOT-CV/jpo-ode/blob/9794fbd3439355f32d645708dc6066d4d7924947/jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimIngestTrackerProperties.java#L14), but the error you're seeing isn't handled correctly in this PR. The work to handle the error you're seeing is [here in the subsequent PR](https://github.com/CDOT-CV/jpo-ode/pull/113/files#diff-f780c6b58b808f60875f60ca9f3e36a1bd5e39546faaf35d0d424b3f2070f219). The code there enables us to catch and log a warning message about empty string variables provided by environment variables and then use the default value for the field instead.

Are we comfortable moving this PR forward with the corrected names and waiting for the upgraded error handling/default values until the next PR? If not, I'm cool with cherry-picking the commit from the following PR and including it in this body of work, but it may be more effort that way.

Reply:
I think that is a reasonable plan. No objections from me.